Library/Basic Set/Basic Set Traits.adq

```diff
@@ -18199,6 +18198,24 @@
 			"base_points": 5,
 			"levels": 1,
 			"points_per_level": 2,
+			"prereqs": {
+				"type": "prereq_list",
+				"all": false,
+				"prereqs": [
+					{
+						"type": "trait_prereq",
+						"has": true,
+						"name": {
+							"compare": "contains",
+							"qualifier": "Language"
+						},
+						"notes": {
+							"compare": "does_not_contain",
+							"qualifier": "None"
```

Comment:
This doesn't check for a language where the character has native literacy as descibed in DF4.  It checks for a language where the character is neither completely illiterate, nor completely incapable of speaking.

Instead of does not contain "None" it should be looking for does contain "Written (Native)"

Reply:
You're right. Fixed.